client/jetpack-connect/remote-credentials.js

```diff
@@ -89,7 +89,7 @@ export class OrgCredentialsForm extends Component {
 		const { translate } = this.props;
 
 		return (
-			<span className="jetpack-connect__install-step">
+			<span className="jetpack-connect__install-step creds-form">
```

Comment:
The `creds-form` class here [needs to be prefixed](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md) with `jetpack-connect__`.